service_provider_config.go

```diff
@@ -40,6 +40,8 @@ const (
 // ServiceProviderConfig enables a service provider to discover SCIM specification features in a standardized form as
 // well as provide additional implementation details to clients.
 type ServiceProviderConfig struct {
+	// PathPrefix is the prefix in the SCIM API URL used to trim the request path in ServeHTTP
```

Comment:
// PathPrefix is the relative URL path from the server root to where this SCIM service
// answers queries.  It is trimmed from paths inside ServeHTTP